web-client/public/js/graph.js

```diff
@@ -93,74 +93,98 @@ var drawGraph = function (nodes, links, positiveWeights, negativeWeights, sheetT
         return Math.round(totalScale(Math.abs(edge.value)));
     };
 
-    var force = d3.layout.force()
-      .size([width, height])
-      .on("tick", tick)
-      .linkDistance($("#linkDistInput").val())
-      .charge($("#chargeInput").val())
-      .chargeDistance($("#chargeDistInput").val())
-      .gravity($("#gravityInput").val());
+    var simulation = d3.forceSimulation()
+      .force("link", d3.forceLink())
+      .force("charge", d3.forceManyBody())
+      .force("center", d3.forceCenter(width / 2, height / 2));
+
+    var drag = d3.drag()
+      .on("start", dragstart)
+      .on("drag", dragged)
+      .on("end", dragended);
```

Comment:
Yes, more like `.force` and `.on` above (though maybe 2 more spaces because our indent size is 4).

Reply:
Applied.